lib/Modules/HyperVCIHooks/HyperVCIHooks.psm1

```diff
@@ -809,6 +809,10 @@ function Initialize-GitRepositories {
     if (($BuildFor -eq "openstack/neutron") -or ($BuildFor -eq "openstack/quantum")) {
         Initialize-GitRepository "$openstackBuild\nova" $NOVA_GIT $BranchName $cherryPicks['nova']
     }
+    if (($BuildFor -eq "openstack/cinder") -or ($BuildFor -eq "openstack/none")) {
```

Comment:
Shouldn't this loop through all repos and IF $buildFor -ne "project" then Initialize-GetRepository...?